scripts/test_all_fuzzers.sh

```diff
@@ -28,6 +28,11 @@ declare -A time_record || (echo "declare -A not avaliable, please update your ba
 # shellcheck disable=SC2116
 for fuzzer in $(echo "$fuzzers" "$backtrace_fuzzers");
 do
+    # skip test for nyx(also skip fmt check, fmt check won't pass in macos)
```

Comment:
What's up with `fmt` on macos?

Reply:
`fmt` needs to build the example, but `libafl_nyx` has no member in `macos`, which will result in a `missing import` error.
It will need some trick to make the example only compile on linux, but in order to make it simple, I just let it go.

Reply:
Should also be doable with `#[cfg(..)]`, I assume?

Reply:
I think `cfg` will add some complexity of the code, and also needs some trick to pass `cargo fmt`, so I write the logic in shellscipt...

Reply:
Better to do it in the actual rust code than in some shell scripts imho :)